plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts

```diff
@@ -81,7 +82,7 @@ export class SessionManager {
         }
     }
 
-    public async add(message: IncomingRecordingMessage): Promise<void> {
+    public async add(message: IncomingRecordingMessage, kafkaMessageTimestamp: number | undefined): Promise<void> {
```

Comment:
This could be made a bit easier by adding it to the "Kafka" part of the incoming recording message instead of as a separate arg?

Reply:
Totally missed that the blob ingester owned that time 👨‍🍳 👌